### ISSUE TITLE
Accessibility improvements - Monaco editor

### ISSUE THREAD

**Issue by dnaidionov** [cardstack/cardstack#1381]
currently there are following failures in themer a11y tests: 

![image](https://user-images.githubusercontent.com/1836666/75270814-2e985680-57c9-11ea-9dfc-7a120ec3a296.png)

```
Elements must have sufficient color contrast 
Violated 8 times. Offending nodes are: 
<div class="line-numbers" style="left:0px;width:36px;">10</div>
<div class="line-numbers" style="left:0px;width:36px;">11</div>
<div class="line-numbers" style="left:0px;width:36px;">12</div>
<div class="line-numbers" style="left:0px;width:36px;">13</div>
<div class="line-numbers" style="left:0px;width:36px;">14</div>
<span class="mtk7">/*&nbsp;This&nbsp;is&nbsp;the&nbsp;isolated&nbsp;base&nbsp;card&nbsp;template.&nbsp;All&nbsp;</span>
<span class="mtk7">cards&nbsp;will&nbsp;inherit&nbsp;this&nbsp;CSS&nbsp;if</span>
<span class="mtk7">no&nbsp;custom&nbsp;CSS&nbsp;is&nbsp;provided&nbsp;for&nbsp;the&nbsp;card.*/</span>
```

**Comment by jenweber** [comment]
Options:
https://microsoft.github.io/monaco-editor/playground.html#customizing-the-appearence-tokens-and-colors

https://github.com/Microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators#2-decide-what-you-want-to-do-wrt-high-contrast
